### ISSUE TITLE
the max number of points

### ISSUE THREAD

**Issue by 1453042287** [traveller59/second.pytorch#137]
![2019-04-05 14-24-45屏幕截图](https://user-images.githubusercontent.com/41471119/55607370-98250600-57ae-11e9-987c-b6576955d612.png)
in this function, dose it mean the max number of points feed to the network is max_points * max_voxels?
what should i change if i have a more dense points? like generated from a depth map? 
thanks a lot!

**Comment by 1453042287** [comment]
BTW, can you provide the detail of function points_to_voxel_3d_np(*args, **kwargs) defined in the spconv_utils.py?

**Comment by traveller59** [comment]
no, the max_voxels define max number of voxels generated from a arbitrary point cloud. you need to increase max_voxels. if you want to use pointpillars or origin VFE architecture, you also need to increase max_points.

The source code of points_to_voxel_3d_np is in ```spconv/include/spconv/point2voxel.h```
